extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -117,28 +117,32 @@ function render_player( $player_data, $attributes ) {
 
 	ob_start();
 	?>
-	<div class="<?php echo esc_attr( $block_classname ); ?>" id="<?php echo esc_attr( $instance_id ); ?>">
-		<noscript>
-			<ol class="jetpack-podcast-player__episodes">
-				<?php foreach ( $player_data['tracks'] as $attachment ) : ?>
-				<li class="jetpack-podcast-player__episode">
-					<a
-						class="jetpack-podcast-player__episode-link"
-						href="<?php echo esc_url( $attachment['link'] ); ?>"
-						role="button"
-						aria-pressed="false"
-					>
-						<span class="jetpack-podcast-player__episode-status-icon"></span>
-						<span class="jetpack-podcast-player__episode-title"><?php echo esc_html( $attachment['title'] ); ?></span>
-						<time class="jetpack-podcast-player__episode-duration"><?php echo ( ! empty( $attachment['duration'] ) ? esc_html( $attachment['duration'] ) : '' ); ?></time>
-					</a>
-				</li>
-				<?php endforeach; ?>
-			</ol>
-		</noscript>
+	<div class="<?php echo esc_attr( $block_classname ); ?> no-js" id="<?php echo esc_attr( $instance_id ); ?>">
+		<ol class="jetpack-podcast-player__episodes">
```

Comment:
Are we mirroring the React generated markup here?

If so should we add a comment which notifies the developer about where and how they can keep the markup in sync.

Reply:
At the moment, not at all. It will come soon with the task "HTML Fallback" on our board and that's when I'd start recommending to mirror all changes as we go.